src/app/pages/applications/forms/chart-wizard.component.ts

```diff
@@ -43,7 +44,7 @@ export class ChartWizardComponent implements OnDestroy {
     this.utils = new CommonUtils();
   }
 
-  createRelations(relations, parentName) {
+  createRelations(relations: any[], parentName: string) {
```

Comment:
@ryousaf The value of "relations" is the array of array, like this [["hostPathEnabled", "=", false], ["datasetName", "=", true]], not the array of object. Should we define new class/type for this array and convert this array to new class/type?

Reply:
Ah, Ok. But in general, we should have a clear scope of what actions are supported by relations and how a relation is represented in the form of a class as I had mentioned. I'm not sure I understand the hierarchy here. We are using relations more and more and we should have a better structure of what a relation object looks like and what actions are supported. Reusability and maintenance will improve, I think.

Reply:
@ryousaf Exactly, we need to represent relations in a class and actually we have already have it `FieldRelation` as well as `RelationGroup`. Both are used for represent relations and its group in other places. In this function the params `relations` is just a raw data from middleware which will be converted to `FieldRelation` class later somewhere else.
I couldn't find the very reason to introduce new class/type here, but anyway made a new class for it as it might be definitely helpful if we have a chance to reuse it somewhere else later.